src/Mscc.GenerativeAI/Types/Generative/FileResource.cs

```diff
@@ -53,5 +53,9 @@ public class FileResource
         /// Output only. The uri of the File.
         /// </summary>
         public string Uri { get; set; }
+        /// <summary>
+        /// Output only. The state of the File.
+        /// </summary>
+        public string State { get; set; }
```

Comment:
This shall be an enumeration of `FileState` to avoid conflict with `State` of tuned model.
Perhaps we can merge both enumerations into a single `State` one specifying a super-set of both status values.